### DISCUSSION TITLE
Getting corrupt feed/green screen on Reolink cameras

### DISCUSSION THREAD

**Discussion by cmer** [bluenviron/mediamtx#438] [Questions]
Whenever I connect rtsp-simple-server to one of my Reolink cameras, I get a corrupted feed like below:

<img width="703" alt="CleanShot 2021-06-04 at 10 44 52@2x" src="https://user-images.githubusercontent.com/412/120820383-7d7f8080-c522-11eb-825f-903865704931.png">

I've seen this happen in the past, and rebooting the camera fixes it, but with rtsp-simple-server, the feed remains corrupted regardless. 

## Which version are you using?

v0.6.12

## Which operating system are you using?

OS

- [X] Linux
- [ ] Windows
- [ ] macOS

Architecture

- [X] amd64
- [ ] arm64
- [ ] arm7
- [ ] arm6

**Comment by cmer** [comment]
Please see Wireshark log: https://gist.github.com/cmer/959996e452c264fa8ca9b5211d754dcb

**Comment by cmer** [comment]
I am able to view the feed correctly with ffmpeg using the following command:

`ffmpeg -i rtsp://user:password@10.48.20.104//h264Preview_01_main -c copy -f rtsp rtsp://127.0.0.1:8554/stream`

Here's the Wireshark log for the ffmpeg proxy feed: https://gist.github.com/cmer/e2945d10299483b3b33fd7aa0874f039

**Comment by cmer** [comment]
I emailed you about this, @aler9. Thanks!

**Comment by jason8745** [comment]
@cmer  I have the same problem. But i solved it by change the protocol . Maybe it's because the UDP protocol . If you want to use proxy mode, you can add sourceProtocol: tcp   in rtsp-simple-server.yml 
```path:
    proxied: 
       source: your ip camera
       sourceProtocol: tcp
```

**Comment by cmer** [comment]
One in a while, a kind Internet soul comes along and just solves your issue. @jason8745 is one of those people. Thank you, it totally did the trick!

I also had to adjust `readBufferSize: 6800`.

**Comment by aler9** [comment]
i'm adding this to the README.

[a comment by an automated account is not shown]
